docs/rst/installation/linux_binary_installation.rst

```diff
@@ -18,12 +65,11 @@ First, add the Qt 5.15 repository, required for the installation of several Fast
     sudo add-apt-repository ppa:beineri/opt-qt-5.15.2-focal
 
 Next, add Vulcanexus GPG key so apt can retrieve the packages:
-
```

Comment:
I suggest keeping this blank line to separate the code block from the sentence and improve readability.